test/common/src/main/kotlin/org/codice/compliance/verification/binding/PostBindingVerifier.kt

```diff
@@ -94,13 +96,18 @@ class PostBindingVerifier(val response: IdpPostResponseDecorator) {
     }
 
     /**
-     * Verifies the destination is correct according to the redirect binding rules in the binding spec
-     * 3.4.5.2 Security Considerations
+     * Verifies the destination is correct according to the post binding rules in the binding spec
+     * 3.5.5.2 Security Considerations
      */
     private fun verifyPostDestination() {
         val destination = response.responseDom.attributes.getNamedItem("Destination")?.nodeValue
-        if (destination != TestCommon.ACS_URL) {
-            throw SAMLComplianceException.createWithPropertyNotEqualMessage(SAMLBindings_3_4_5_2_a1,
+        val signatures = response.responseDom.allChildren("Signature")
+
+        // "If the message is signed, the Destination XML attribute in the root SAML element of the
```

Comment:
✏️ I'm not sure I see a ton of value putting a comment here that has the same text as the message to be returned.

Reply:
Agreed

Reply:
👍